index.html

```diff
@@ -53,5 +53,142 @@ <h1 id="myName">
+                <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                <ul class="card-techskills">
+                  <li class="card-skillset">HTML</li>
+                  <li class="card-skillset">Bootstrap</li>
+                  <li class="card-skillset">RUBY</li>
+                </ul>
+                <button class="card-btn">See project</button>
+              </div>
+              <div class="card-works">
+                <h2 class="card-title">Profesional Art Printing Data</h2>
+                <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                <ul class="card-techskills">
+                  <li class="card-skillset">HTML</li>
+                  <li class="card-skillset">Bootstrap</li>
+                  <li class="card-skillset">RUBY</li>
+                </ul>
+                <button class="card-btn">See project</button>
+              </div>
+              <div class="card-works">
+                <h2 class="card-title">Profesional Art Printing Data</h2>
+                <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                <ul class="card-techskills">
+                  <li class="card-skillset">HTML</li>
+                  <li class="card-skillset">Bootstrap</li>
+                  <li class="card-skillset">RUBY</li>
+                </ul>
+                <button class="card-btn">See project</button>
+              </div>
+              <div class="card-works">
+                <h2 class="card-title">Profesional Art Printing Data</h2>
+                <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                <ul class="card-techskills">
+                  <li class="card-skillset">HTML</li>
+                  <li class="card-skillset">Bootstrap</li>
+                  <li class="card-skillset">RUBY</li>
+                </ul>
+                <button class="card-btn">See project</button>
+              </div>
+            </div>
+        </div>
+      </section>
+      <section class="about">
+        <div class="about-me">
+          <h2 class="about-title">About me</h2>
+          <p class="about-text">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+          <button class="resume">Get my resume</button>
+        </div>
+         <div class="about-img"><img src="img/Frame.png"> </div> 
+         <hr class="divider">
+      </section>
+      <section class="profession-tools">
```

Comment:
- [ ] I would suggest you please stick to the design as much as possible like spacing, color, and font. The content is in the middle of the page. Please adjust the spacing.
- [ ] I notice that your paragraph is different from design please use the same font you can use from google font.
- [ ] Decorations are always backgrounded images, not `img` tags please use the Frame.png as background image not img tag.
- [ ] Also please work on responsiveness as the project is not responsive.
### Your design
![his](https://user-images.githubusercontent.com/81727988/236489511-e5af1c3c-06da-4ce0-a6da-4d1bf010b9b0.png)

### Figma design
![fimga](https://user-images.githubusercontent.com/81727988/236489641-7fc60ba8-532c-4aa1-b3f6-466e547bda1c.png)